Bootloaders/HID/BootloaderHID.c

```diff
@@ -146,22 +146,23 @@ void EVENT_USB_Device_ControlRequest(void)
 			while (!(Endpoint_IsOUTReceived()));
 
 			/* Read in the write destination address */
-			#if (FLASHEND > 0xFFFF)
+			#if (FLASHEND > USHRT_MAX)
 			uint32_t PageAddress = ((uint32_t)Endpoint_Read_16_LE() << 8);
 			#else
 			uint16_t PageAddress = Endpoint_Read_16_LE();
 			#endif
 
 			/* Check if the command is a program page command, or a start application command */
-			#if (FLASHEND > 0xFFFF)
+			#if (FLASHEND > USHRT_MAX)
```

Comment:
@abcminiuser Why didnt you pickup this line of fix?

Reply:
They're equivalent, no? I prefer the explicit constant since the constraint isn't the type but the device's actual memory layout, but I suppose the named value works just as well.

Reply:
Yes, just keep it. Its fine and should be better.